english/Harpers/yourlife.md

```diff
@@ -0,0 +1,5 @@
+This is the beginning of your life. Make wise choices to cultivate a healthy body
+and a healthy mind.
+
+click [here](https://github.com/udacity/create-your-own-adventure/blob/master/Polish/przygoda/jedzenie/akap/drzwi/drzwi.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.